oidc-api/src/main/java/uk/gov/di/authentication/oidc/services/UserInfoService.java

```diff
@@ -129,6 +129,10 @@ private void populateInfoOrchSplitEnabled(UserInfo userInfo, AccessTokenInfo acc
                     AuthUserInfoClaims.PUBLIC_SUBJECT_ID.getValue(),
                     tmpUserInfo.getClaim(AuthUserInfoClaims.PUBLIC_SUBJECT_ID.getValue()));
         }
+        if (accessTokenInfo.getScopes().contains(CustomScopeValue.WALLET_SUBJECT_ID.getValue())) {
+            var walletSubjectID = calculateWalletSubjectID(accessTokenInfo);
+            userInfo.setClaim("wallet_subject_id", walletSubjectID);
```

Comment:
Still a bit confusing having it set as a claim when returning it

Reply:
Discussed the difference between claim and requested claim